basics/installation/install-with-docker.md

```diff
@@ -0,0 +1,471 @@
+
+Complete docker compose manifest for reference:
+
+```yaml
+version: '3'
+services:
+  mysql:
+    container_name: some-mysql
+    image: mysql:5.7
+    restart: unless-stopped
+    environment:
+      MYSQL_ROOT_PASSWORD: admin
+      MYSQL_DATABASE: prestashop
+    networks:
+      - prestashop_network
+  prestashop:
+    container_name: prestashop
+    image: prestashop/prestashop:latest
+    restart: unless-stopped
+    depends_on:
+      - mysql
+    ports:
+      - 8080:80
+    environment:
+      DB_SERVER: some-mysql
+      DB_NAME: prestashop
+      DB_USER: root
+      DB_PASSWD: admin
+      PS_DEMO_MODE: 1
+      PS_INSTALL_AUTO: 1
+      PS_DOMAIN: localhost:8080
+    networks:
+      - prestashop_network
+networks:
+    prestashop_network:
+```
+
+### Manually install PrestaShop with Installation Assistant, and begin testing
+
+To manually install PrestaShop using the Installation Assistant, don't set the environment variables `PS_DEMO_MODE` and `PS_INSTALL_AUTO`.
+
+In the directory where the docker compose.yml file is located, run: 
+
+- Start the stack: 
+
+```
+docker compose up
+```
+
+- Start the stack, in background: 
+
+```
+docker compose up -d
+```
+
+Access `http://localhost:8080` on your browser to begin testing PrestaShop. 
+
+You will land on the installer of PrestaShop. 
+
+- When you will be asked for MySQL server, type `some-mysql`, as declared in your yaml file.
+- When asked for MySQL user, use `root`, as declared in your yaml file.
+- When asked for MySQL password, use `admin`, as declared in your yaml file.
+
+#### Installation assistant and admin URLs
+
+{{% notice note %}}
+It is a grood practice to change `install/` and `admin/` directory names. 
+
+To achieve this, specify the two following environment variables in your PrestaShop container declaration:
+
+```
+PS_FOLDER_ADMIN: admin4577
+PS_FOLDER_INSTALL: install4577
+```
+
+In this example, we set them to `admin4577` and `install4577`. When installing PrestaShop with Docker for the first time, you need to access `https://localhost:8080/install4577`.
+{{% /notice %}}
+
+{{% notice info %}}
+If you did not change the `install/` or `admin/` directory names, after installation, you need to manually delete the install directory in your container, and change the admin directory name. 
+To do this, please run: 
+
+```
+docker exec -i prestashop rm -rf install
+docker exec -i prestashop mv admin admin_xxx
+```
+Your admin is now located at `https://localhost:8080/admin_xxx`. 
+{{% /notice %}}
+
+### Bind a volume for Mysql persisting
+
+If you don't bind a volume to the Mysql container, the database content will not be persisted when stopping / restarting the stack.
+
+To avoid this, add a `volume` to your Mysql container: 
+
+```yml
+...
+  mysql:
+    container_name: some-mysql
+    volumes:
+      - dbdata:/var/lib/mysql
+...
+volumes:
+  dbdata:
+```
+
+### Bind a volume for Installation persisting
```

Comment:
```suggestion
### Bind a volume for installation persisting
```
I don't like the title. I understand that it is to persist entire instance of the PrestaShop, maybe the word "instance" fits better there?